lisa/sut_orchestrator/azure/features.py

```diff
@@ -170,6 +170,16 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         super()._initialize(*args, **kwargs)
         self._initialize_information(self._node)
 
+    def _get_primary(self, nics: Any) -> Any:
```

Comment:
Don't use Any here, it breaks the benefit of typing. Try to use the type with quote, and import it only when type checking.

Reply:
updated